xml/System.Data.Common/DbDataReader.xml

```diff
@@ -2190,17 +2133,9 @@ private static void GetCredits(String connectionString) {
       </Parameters>
       <Docs>
         <param name="ordinal">The zero-based column to be retrieved.</param>
-        <summary>An asynchronous version of <see cref="M:System.Data.Common.DbDataReader.IsDBNull(System.Int32)" />, which gets a value that indicates whether the column contains non-existent or missing values.</summary>
+        <summary>Asynchronously gets a value that indicates whether the column contains non-existent or missing values.</summary>
```

Comment:
I would leave this out because it feels unnecessary. Both the ordinal and the name (used in the extension methods on DataReaderExtensions) are just ways to identify the column.